sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -17,6 +22,38 @@
 from ._user_agent import USER_AGENT
 
 
+_SERIALIZER = Serializer()
+_SERIALIZER.client_side_validation = False
+
+
+def _format_api_version(request: HttpRequest, api_version: str) -> HttpRequest:
+    """Returns a request copy that includes an api-version query parameter if one wasn't originally present.
+
+    :param request: The HTTP request being sent.
+    :type request: :class:`~azure.core.rest.HttpRequest`
+    :param str api_version: The service API version that the request should include.
+
+    :returns: A copy of the request that includes an api-version query parameter.
+    :rtype: :class:`~azure.core.rest.HttpRequest`
+    """
+    request_copy = deepcopy(request)
+    params = {"api-version": api_version}  # By default, we want to use the client's API version
```

Comment:
Please note that the v2 api versions (v2.0 and v2.1) were part of the path. Here is an example recording for you to see how one of those requests looks: https://github.com/Azure/azure-sdk-assets/blob/311c87465f5791be02af4225175e876ec4087d4e/python/sdk/formrecognizer/azure-ai-formrecognizer/tests/recordings/test_frc_invoice_from_url.pyTestInvoiceFromUrltest_invoice_locale_specified.json#L4

For v2 api version support we should check the path for an existing api version and respect that. If you're on one of the older clients (i.e `FormRecognizerClient` or `FormTrainingClient`) you'll have to make sure that the api version is added correctly in the path. It would be good to have tests for that scenario.

Reply:
Thanks @catalinaperalta for your review! 
Yup, we do not support v2 api versions in this release. I logged an issue for it: https://github.com/Azure/azure-sdk-for-python/issues/32366 Will make it available in next release.